katib-db-manager/rockcraft.yaml

```diff
@@ -0,0 +1,60 @@
+# Based on https://github.com/kubeflow/katib/blob/v0.15.0/cmd/katib-controller/v1beta1/Dockerfile
+name: katib-db-manager
+summary: Katib DB Controller
+description: |
+  Katib database manager.
+version: v0.15.0_22.04_1
+license: Apache-2.0
+build-base: ubuntu:22.04
+base: bare
+run-user: _daemon_
+services:
+  katib-db-manager:
+    override: replace
+    summary: "katib-db-manager service"
+    startup: enabled
+    command: "/app/katib-db-manager"
```

Comment:
I think using a working-dir in each of your ROCKs is a good practice if possible